srcdeps-maven-plugin/src/main/java/org/srcdeps/mvn/plugin/ExtensionsXmlUtils.java

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright 2015-2017 Maven Source Dependencies
+ * Plugin contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.srcdeps.mvn.plugin;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.Text;
+
+/**
+ * Static methods to modify {@code extensions.xml} DOM.
```

Comment:
Why, where? I see down there that it "updates srcdeps", so, this seems important enough to me to deserve a better javadoc :)